scripts/extract_video_frames.py

```
@@ -0,0 +1,155 @@
+        "ffmpeg not found. Please install ffmpeg:\n"
+        "  Option 1: choco install ffmpeg\n"
+        "  Option 2: Download from https://ffmpeg.org/download.html\n"
+        "  Option 3: Add ffmpeg to PATH"
+    )
+
+
+def extract_frames_ffmpeg(
+    video_file: str,
+    output_dir: str,
+    frame_interval: int = 10,
+    crop: str = None,
```

Ruff (0.15.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)